requirements.txt

```
@@ -0,0 +1,48 @@
+###
+### Third-party
+###
+cython
+msgpack==0.5.6
+numexpr
+packaging
```

```suggestion
packaging>=20.0
```

See https://github.com/pupil-labs/pupil/releases/v2.4 for reference.